translations/messages+intl-icu.en.yaml

```diff
@@ -84,14 +84,14 @@ team.click_on_member: 💡 Click on a member to discover their full profile.
 team.modal.nice: Nice!
 team.reviewers: Reviewers
 team.review.twig_alias_1: Could we alias this import to <0>Twig</0> to make it a little more understandable?
-team.review.twig_alias_2: Another idea could be to extend the <0>AbstractController</0> and use the <0>render</0> method, WDYT?
-team.review.infra_1: Always use interfaces instead of concrete implementations.<0/>It takes advantage of <1>polymorphism</1> and reduce coupling between classes.
-team.review.infra_2: 👍 Plus, the <0>MemberRepository</0> belongs to the "Domain" layer.<1/>We must rely as much as possible on code from that layer.
```

Comment:
Making Mathias looks more kind 😁  "must" is a strong imperative that should generally be avoided in code reviews. Relying as much as possible on the Domain layer it is not an end in itself also, especially in the infra